packages/constants/tsup.config.ts

```diff
@@ -0,0 +1,12 @@
+import { defineConfig } from 'tsup';
+
+export default defineConfig({
+  entry: ['src/index.ts'],
+  dts: true,
+  format: 'esm',
+  target: 'es2023',
```

Comment:
Did this end up being necessary? I assumed that it was using the same target from the tsconfig

Reply:
Indeed, I was made aware of this fact after making this change so this is definitely redundant